apps/synapse-oauth-signin/src/OAuth2Form.tsx

```diff
@@ -139,7 +140,9 @@ export function OAuth2Form() {
     if (scope == null) {
       missingParams.push('scope')
     }
-    const responseType = searchParams.get('response_type')
+    const responseType = searchParams.get(
+      'response_type',
+    ) as OIDCAuthorizationRequestResponseTypeEnum
```

Comment:
The fixes in the OpenAPI spec changed a lot of these generated properties from `string` type to now reference an enumeration, so we use the enumeration or add casts to fix new type errors.